controllers/tykoasapidefinition_controller.go

```diff
@@ -138,10 +154,25 @@ func (r *TykOasApiDefinitionReconciler) Reconcile(ctx context.Context, req ctrl.
 		return ctrl.Result{}, nil
 	}
 
+	if tykOAS.Status.IsVersionedAPI {
+		err := r.handleVersionedTykOASAPI(ctx, log, tykOAS)
```

Comment:
what do you think renaming this method name (`handleVersionedTykOASAPI`) to align its name with other methods; for instance, `reconcileVersionedOasApi` or sth like this.